### ISSUE TITLE
win32eventreactor docstring should link to the Process docstring it mentions.

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4624]
|<img alt="Screwtape's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Screwtape reported|
|-|-|
|Trac ID|trac#4624|
|Type|enhancement|
|Created|2010-08-04 10:03:52Z|
|Branch|https://github.com/twisted/twisted/tree/link-to-Process-docs-4624|

The `win32eventreactor` docstring says "Process running has some problems (see Process docstring)." but it doesn't link to the Process docstring in question. It's not at all obvious which Process is being referred to without source-diving.

<details><summary>Searchable metadata</summary>

```
trac-id__4624 4624
type__enhancement enhancement
reporter__Screwtape Screwtape
priority__normal normal
milestone__ 
branch__branches_link_to_Process_docs_4624 branches/link-to-Process-docs-4624
branch_author__screwtape screwtape
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1280916232000000 1280916232000000
changetime__1302663433000000 1302663433000000
version__None None
owner__Screwtape Screwtape

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="screwtape's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_1"></a>|screwtape commented|
|-|-|

(In [29909]) Create branch link-to-Process-docs-4624

**Comment by twisted-trac** [comment]
|<img alt="screwtape's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|screwtape commented|
|-|-|

(In [29910]) Link to Process docs.

Refs: [#4624](https://github.com/twisted/twisted/issues/4624)

**Comment by twisted-trac** [comment]
|<img alt="screwtape's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|screwtape commented|
|-|-|

(In [29911]) Link to Process docs.

Refs: [#4624](https://github.com/twisted/twisted/issues/4624)

**Comment by twisted-trac** [comment]
|<img alt="screwtape's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|screwtape commented|
|-|-|

(In [29912]) Revert r29910

Didn't mean to commit this to trunk, sorry. :(

Refs: [#4624](https://github.com/twisted/twisted/issues/4624)

**Comment by twisted-trac** [comment]
|<img alt="Screwtape's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_5"></a>|Screwtape removed owner|
|-|-|

OK, I accidentally committed this to trunk in r29910, but I reverted it in r29912. In that brief time however, builders were triggered and [the resulting API docs](http://buildbot.twistedmatrix.com/builds/apidocs-29910.tar.bz2) look good to me.

r29911 is the same change, committed on the branch.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set owner to Screwtape|
|-|-|

Needs a `4624.misc`, otherwise looks good.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_7"></a>|@exarkun set status to `closed`|
|-|-|

(In [31654]) Merge link-to-Process-docs-4624

Author: Screwtape
Reviewer: exarkun
Fixes: [#4624](https://github.com/twisted/twisted/issues/4624)

Add an API doc link from the win32eventreactor module docstring to the primary
process-support implementation class.
